FreeRTOS/FreeRTOS
# Update wolfSSL to the latest version(v.4.5.0)

Update wolfSSL to the latest version(v.4.5.0)

Description
-----------
This demo contains the following updates:
- wolfSSL has been updated to v.4.5.0
- The certificate used in this demo has an extended validity period.
- The default TLS protocol version used has changed to v1.3


Test Steps
-----------
Build the solution then run with debugger.

Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
